### PR TITLE
Fix particle disruptor ammo

### PR DESCRIPTION
**Pull request by Vladislav-CS**

### CODE REVIEW

File: EXILED/Exiled.API/Features/Pickups/FirearmPickup.cs [hunk 1]
```diff
@@ -37,6 +37,9 @@ internal FirearmPickup(ItemType type)
         {
             Base = (BaseFirearm)((Pickup)this).Base;
             IsDistributed = true;
+
+            if (type is ItemType.ParticleDisruptor && Status.Ammo <= 0)
+                Status = new FirearmStatus(5, FirearmStatusFlags.MagazineInserted, 0);
```
Comment (louis1706) [inline EXILED/Exiled.API/Features/Pickups/FirearmPickup.cs:42]: can you explain why it's needed ?

Comment (Vladislav-CS) [reply EXILED/Exiled.API/Features/Pickups/FirearmPickup.cs:42]: If you spawn an Particle Disruptor pickup, it will be immediately destroyed as no ammo

**Review by louis1706 — APPROVED** [review]

File: EXILED/Exiled.API/Features/Pickups/FirearmPickup.cs [hunk 2]
```diff
@@ -37,6 +37,9 @@ internal FirearmPickup(ItemType type)
         {
             Base = (BaseFirearm)((Pickup)this).Base;
             IsDistributed = true;
+
+            if (type is ItemType.ParticleDisruptor && Status.Ammo <= 0)
```
Comment (Misfiy) [inline EXILED/Exiled.API/Features/Pickups/FirearmPickup.cs:41]: Replace <= with ==

It is byte, with min value of 0

Comment (Misfiy) [inline EXILED/Exiled.API/Features/Pickups/FirearmPickup.cs:41]:
```suggestion
            if (type is ItemType.ParticleDisruptor && Status.Ammo == 0)
```

**Review by Misfiy — APPROVED** [review]

**Review by Jesus-QC — APPROVED** [review]

**Pull request merged**